specs/_features/eip7594/p2p-interface.md

```diff
@@ -64,16 +65,30 @@ class DataColumnIdentifier(Container):
 
 ### Helpers
 
-##### `verify_data_column_sidecar_kzg_proofs`
+##### `verify_data_column_sidecar`
 
 ```python
-def verify_data_column_sidecar_kzg_proofs(sidecar: DataColumnSidecar) -> bool:
+def verify_data_column_sidecar(sidecar: DataColumnSidecar) -> bool:
     """
-    Verify if the proofs are correct.
+    Verify if the data column sidecar is valid.
     """
+    # A sidecar for zero blobs is invalid
+    assert len(sidecar.kzg_commitments) > 0
```

Comment:
do you want to convert these to `bool` conditions, rather than asserts?

Reply:
Yeah I had that thought too. I'll do that.

Reply:
The asserts were prettier, but I suppose this is the right thing to do.